sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs

```diff
@@ -1594,6 +1594,63 @@ public void Cfg_Coalesce_MultipleAssignments()
             simpleBlockWithFullExpression.Instructions.Should().ContainSingle("a = a ?? (b = b ?? c)");
         }
 
+        [TestMethod]
+        [TestCategory("CFG")]
+        public void Cfg_Coalesce_Throw()
```

Comment:
It's a bit strange that we have these changes in this commit. Can we rebase the feature branch?

Reply:
good point, I rebased on master as the feature branch is not relevant anymore